content/pt-br/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,528 @@
+{{< glossary_definition term_id="ingress" length="all" >}}
+
+
+<!-- body -->
+
+## Terminologia
+
+Para mais clareza, este guia define os seguintes termos:
+
+* Nó: Uma máquina de trabalho no Kubernetes, parte de um cluster.
+* Cluster: Um conjunto de Nós que executam aplicações em contêiner gerenciado pelo Kubernetes. Para este exemplo, e nas implantações mais comuns do Kubernetes, os Nós no cluster não fazem parte da Internet pública.
+* Roteador de borda: Um roteador que impõe a política de firewall para o seu cluster. Isso pode ser um gateway gerenciado por um provedor de nuvem ou um  hardware físico.
+* Rede do cluster: Um conjunto de links, lógicos ou físicos, que facilitam a comunicação dentro de um cluster de acordo com o [modelo de rede](/docs/concepts/cluster-administration/networking/) do Kubernetes.
+* Serviço: Um {{< glossary_tooltip text="serviço" term_id="service" >}} Kubernetes que identifica um conjunto de Pods usando a {{< glossary_tooltip text="label" term_id="label" >}} selectors. Salvo indicação em contrário, assume-se que os Serviços tenham IPs virtuais apenas roteáveis dentro da rede de cluster.
+
+
+## O que é o Ingress?
+
+O [Ingress](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#ingress-v1-networking-k8s-io) expõe rotas HTTP e HTTPS de fora do cluster para um {{< link text="serviço" url="/docs/concepts/services-networking/service/" >}} dentro do cluster.
+O roteamento do tráfego é controlado por regras definidas no recurso Ingress.
+
+Aqui está um exemplo simples em que o Ingress envia todo o seu tráfego para um serviço:
+
+{{< figure src="/docs/images/ingress.svg" alt=" diagrama do Ingress" class="diagram-large" caption="Figura. Ingress" link="https://mermaid.live/edit#pako:eNqNkstuwyAQRX8F4U0r2VHqPlSRKqt0UamLqlnaWWAYJygYLB59KMm_Fxcix-qmGwbuXA7DwAEzzQETXKutof0Ovb4vaoUQkwKUu6pi3FwXM_QSHGBt0VFFt8DRU2OWSGrKUUMlVQwMmhVLEV1Vcm9-aUksiuXRaO_CEhkv4WjBfAgG1TrGaLa-iaUw6a0DcwGI-WgOsF7zm-pN881fvRx1UDzeiFq7ghb1kgqFWiElyTjnuXVG74FkbdumefEpuNuRu_4rZ1pqQ7L5fL6YQPaPNiFuywcG9_-ihNyUkm6YSONWkjVNM8WUIyaeOJLO3clTB_KhL8NQDmVe-OJjxgZM5FhFiiFTK5zjDkxHBQ9_4zB4a-x20EGNSZhyaKmXrg7f5hSsvufUwTMXThtMWiot5Jh6p9ffimHijIezaSVoeN0uiqcfMJvf7w" >}}
+
+Um Ingress pode ser configurado para fornecer URLs acessível externamente aos serviços, balanceamento de carga de tráfego, encerrar SSL/TLS e oferecer hospedagem virtual baseada em nome.
+Um [controlador Ingress](/docs/concepts/services-networking/ingress-controllers) é responsável por atender o Ingress, geralmente com um balanceador de carga, embora também possa configurar seu roteador de borda ou frontends adicionais para ajudar a lidar com o tráfego.
+
+Um Ingress não expõe portas ou protocolos arbitrários. 
+Expor serviços à Internet que não sejam HTTP e HTTPS normalmente usa um serviço do tipo [Service.Type=NodePort](/docs/concepts/services-networking/service/#type-nodeport) ou [Service.Type=LoadBalancer](/docs/concepts/services-networking/service/#loadbalancer).
```

Comment:
I feel this sentence could be revamped. Perhaps something along the lines of "Para expor ... normalmente um serviço do tipo ... é utilizado".

What do you think?

Reply:
Look again, please.